frontend/src/pages/RecipesPage.tsx

```diff
@@ -7,19 +7,22 @@ type RecipesPageProps = {
 
 export default function RecipesPage(props: Readonly<RecipesPageProps>) {
     return (
-        <div className="recipe-list">
-            <h2>Recipe List</h2>
-            <div className="recipes-container" style={{display: 'flex', flexWrap: 'wrap'}}>
-                {props.recipes.map((recipe) => (
-                    <RecipeCard key={recipe.id} recipe={recipe}/>
-                ))}
+        <div className="recipes-page-container"> {/* Wrapper div */}
+            <div className="recipe-list">
+                <h2>Recipe List</h2>
+                <div className="recipes-container">
+                    {props.recipes.map((recipe) => (
+                        <RecipeCard key={recipe.id} recipe={recipe}/>
+                    ))}
+                </div>
+                <br/>
+                <br/>
+                <br/>
+                <br/>
+                <br/>
+                <br/>
+                <br/>
```

Comment:
Couldn't you give the container a margin: bottom instead of the several <br/> ?

Reply:
Im working on it. Its a bit tricky, when the you have a footer that appears only at the bottom of the page. :)